l10n_it_website_portal_fiscalcode/views/l10n_it_website_portal_fiscalcode_templates.xml

```diff
@@ -7,19 +7,15 @@
     <template
         id="fiscalcode_details"
         name="Add fiscalcode"
-        inherit_id="portal.portal_my_details"
+        inherit_id="portal.portal_my_details_fields"
     >
-        <xpath
-            expr="//div[hasclass('o_portal_details')]/div[1]/div[1]/div[7]"
-            position="after"
-        >
-            <!-- After VAT -->
+        <xpath expr="//div[label[@for='vat']]" position="after">
             <div
                 t-attf-class="form-group #{error.get('fiscalcode') and 'o_has_error' or ''} col-xl-6"
             >
                 <label
                     class="col-form-label label-optional"
-                    for="vat"
+                    for="fiscalcode"
                 >Fiscal Code</label>
                 <input
                     type="text"
```

Comment:
Più sotto negli attributi `t-att*` si usano logiche copiate da come era fatto il template in `14.0`, queste sono però state corrette in `16.0` (vedi https://github.com/odoo/odoo/commit/beb50e8047f4dbe148933edd52cceadb2d5cccde); è possibile riportare le correzioni anche in questa migrazione?

Reply:
:+1: